Xamarin.Forms.Core/MultiTapPressGestureRecognizer.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Xamarin.Forms
+{
+	public class MultiTapPressGestureRecognizer : GestureRecognizer
+	{
+		public static readonly BindableProperty CommandProperty =
```

Comment:
Can't we share these base commands ? like a CommandGestureRecognizer ?

Reply:
I think that different gestures can rise different commands. so in the base class there is no need to make commands.